batch_convert.qml

```diff
@@ -455,6 +494,7 @@ MuseScore {
     exportExcerpts.checked = false
     // 'uncheck' everything, then 'check' the next few
     inMscz.checked = outPdf.checked = true
+    differentExportPath.checked = false
```

Comment:
what about resetting `ePath` and `iPath`?

Reply:
Currently the fileDialog opens the first time in $ProgramFiles/MuseScore3/QmlThings. Which in any case is not a good place. Maybe the default should be $documents/MS/scores?. Or simply $documents...

Reply:
Documents/Musescore3/scores make sense

Reply:
But reset defaults applies to the ui. The sources folder and the export folder are not in the layout. So the user won't know it has been reset.
Plus I don't see a case where the user would be disturbed to have the dialogs open where they previously opened.

Reply:
I set the default to [this](https://github.com/musescore/MuseScore/blob/master/mscore/preferences.h#L88)

Reply:
yes, for iPart and ePath.